Himanshu_Kumar/project(Profile)/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+ 
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <style type="text/css">
+	.navbar div a{
+		color: white;
+	}
+	.navbar-brand{
+		font-size: 25px;
+		 font-family:Comic Sans MS;
+	}
+    </style>
+</head>
+<body style="background-image: url(images/Backgrounds.jpg)">
+
+
+    <div class="row" style="margin-top: 50px;">
+        <div class="col-lg-4 col-lg-offset-4">
+            <div class="panel panel-default" style="border: 0px; border-radius: 7px;">
+              <div class="panel-heading" style="background-color:#6AAD4A; font-size: 20px;
+             font-family:Comic Sans MS; text-align: center; color: white;">
+                    <strong>My Login Page</strong>	
+              </div>
+              <div class="panel-body">
+               <img src="images/HK_image.jpg" style="width: 100px; height: 100px; margin-left: 37%; margin-bottom: 20px;">
+                  <form class="form-horizontal" action="#" style="margin-bottom: 20px;">
+                    
+                        <div class="form-group">
+                            <label for="inputEmail3" class="col-lg-2 control-label">Usename</label>
+                            <div class="col-lg-10">
+                                <input type="text" class="form-control" name="UName" placeholder="Enter Username" required="required"></input>
+                            </div>
+                        </div>
+                        <div class="form-group">
+                            <label for="inputEmail3" class="col-lg-2 control-label">Password</label>
+                            <div class="col-lg-10">
+                                <input type="password" class="form-control" name="Pass" placeholder="Enter Password" required="required"></input>
+                            </div>
+                        </div>
+                        <button class="btn" type="submit" name="submit" style="margin-left: 44%; margin-bottom: 10px;">Log In</button>
+                        <br>
+                        <a href="signup.html" style="font-size: 12px; text-decoration: underline;">New User? Sign Up</a>
```

Comment:
Since you have used interal css above it can be utilised for the <a> tags as most of the styling is same.

Reply:
a small addition, why even using internal CSS, this is a clear violation of the Single Responsibility Principle

Reply:
I understand that..... That was totally an old file which I replaced and my original .CSS file doesn't contain any of that.